src/main/scala/com/typesafe/sbt/packager/archetypes/AshScriptPlugin.scala

```diff
@@ -73,15 +73,14 @@ import SbtNativePackager.{ Universal, Debian }
  */
 object AshScriptPlugin extends AutoPlugin {
 
-  val bashTemplate = "bash-template"
-
   override def requires = JavaAppPackaging
 
-  //object autoImport extends JavaAppKeys
-
   import JavaAppPackaging.autoImport._
 
+  val ashTemplate = "ash-template"
+
   override def projectSettings = Seq(
+    bashScriptTemplateLocation := (sourceDirectory.value / "templates" / ashTemplate),
```

Comment:
I fear this won't be enough. You have to create the _ash template_. Something like

bashScriptTemplateLocation := {
   // generate the file, so it can be found
   val script = (target in Universal) / "tmp" / "ash-start-script"
   val content = .. // getResource from internal 
   // fill in the contents from the interally shipped jar
   IO.write(script, content)
   // return this as template location
   script
}